org.adempiere.base/src/org/compiere/model/MSysConfig.java

```diff
@@ -236,6 +236,8 @@ public class MSysConfig extends X_AD_SysConfig
     public static final String ZK_DASHBOARD_PERFORMANCE_TIMEOUT = "ZK_DASHBOARD_PERFORMANCE_TIMEOUT";
     public static final String ZK_DASHBOARD_REFRESH_INTERVAL = "ZK_DASHBOARD_REFRESH_INTERVAL";
     public static final String ZK_DECIMALBOX_PROCESS_DOTKEYPAD = "ZK_DECIMALBOX_PROCESS_DOTKEYPAD";
+	public static final String ZK_DESKTOP_BACKGROUND_COLOR = "ZK_DESKTOP_BACKGROUND_COLOR"; // used in desktop.css.dsp
+	public static final String ZK_DESKTOP_BACKGROUND_MESSAGE_VALUE = "ZK_DESKTOP_BACKGROUND_MESSAGE_VALUE";

```

Comment:
I think the naming of this 2 constant is misleading, it is really just the desktop_header.

Reply:
> I think the naming of this 2 constant is misleading, it is really just the desktop_header.

done, renamed to ZK_DESKTOP_HEADER_BACKGROUND_COLOR and ZK_DESKTOP_HEADER_MESSAGE_VALUE